README.md

```diff
@@ -123,5 +123,146 @@ LabelCell(key: "company", state: LabelState(text: "Shopify",
 
 At the end of the day `HostCell` is just one of the possible implementations of `CellConfigType`, that's the underlying power of this framework.
 
+### Building custom Cells
+
+In order to build custom cells we will need to have 3 parts:
+
+1. A custom view class
+2. A class to manage the view state
+3. A list builder that defines the `HostCell<View, State, Layout>`
```

Comment:
We don't need this step, this is just the convention we have been using

Reply:
I can put is as optional?